<?xml version="1.0"?>
<launch>
  <arg name="cloud_in" default="cam3d/depth_registered/points"/>

  <node pkg="cob_table_object_cluster" type="table_object_cluster_node" name="table_object_cluster" output="screen" >
    <remap from="/get_point_map" to="/point_map/get_map"/>
    <remap from="point_cloud" to="$(arg cloud_in)"/>
    <remap from="shape_array" to="supporting_plane_extraction/shape_array_pub"/>
    <remap from="cluster_array" to="tabletop_object_cluster/cluster_array"/>

    <param name="file_path" type="string" value="/home/goa/tmp/"/>
    <param name="save_to_file" type="bool" value="false"/>
    <!-- height is relative to in normal direction of the plane,
    and the normal is always flipped towards map origin, so negative z. -->
    <param name="height_min" type="double" value="-0.5"/>
    <param name="height_max" type="double" value="-0.03"/>
    <param name="min_cluster_size" type="int" value="10"/>
    <param name="cluster_tolerance" type="double" value="0.06"/>
    <param name="enable_action_mode" type="bool" value="true"/>
  </node>
</launch>